S3 Storage for pdf

herrnesto opened the pull request.

herrnesto committed:
Update requirements.txt

herrnesto committed:
Update settings.py

herrnesto committed:
Update settings.py

herrnesto committed:
Update settings.py

herrnesto committed:
Update settings.py

herrnesto committed:
Update settings.py

herrnesto committed:
Update requirements.txt

herrnesto committed:
Update requirements.txt

herrnesto committed:
Update settings.py

Commit:
add custom storage

Commit:
changed settings for s3

Commit:
fix typo

herrnesto committed:
chnage file storage

herrnesto committed:
fix bug

herrnesto committed:
renamed custom storage module

herrnesto committed:
renamed class

mbschenkel commented on bioco/settings.py:
```diff
@@ -220,7 +220,30 @@ def whitelist_email_from_env(var_env_name):
+                    'AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_STORAGE_BUCKET_NAME')
+
+  # Default Django Storage API behavior - don't overwrite files with same name
+AWS_S3_FILE_OVERWRITE = False
```
Does this mean we keep all the old versions? Or how does that behave?

mbschenkel committed:
Merge branch 'main' into storage

# Conflicts:
#	requirements.txt

mbschenkel committed:
Grouping AWS dependent settings

Grouping AWS dependent settings behind env variable check to make local testing easier without requiring AWS
